recipes/betse/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "betse" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "845896f4c30b3970d5bd9501e0ca754eef5af0d742de76103f66c8c8c58e4eca" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - betse = betse.__main__:main
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<34]
```

Comment:
You don't need this line, and this can be noarch with a python version constraint.

Reply:
I *thought* that might be superfluous. I shall make it so – and, as you suggest, enable `noarch`.

A kitten for your troubles, Good Sir. :cat2: